src/main/java/org/threadly/concurrent/PriorityScheduler.java

```diff
@@ -573,65 +563,16 @@ public void run() {
      */
     protected TaskWrapper getNextReadyTask() throws InterruptedException {
```

Comment:
This is where the major logic change in PriorityScheduler is.  We get the task from AbstractPriorityScheduler, then check if we should park or attempt to execute the task.